recipes/ditaa/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ditaa" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stathissideris/ditaa/releases/download/v0.11.0/ditaa-0.11.0-standalone.jar
+  sha256: 9418aa63ff6d89c5d2318396f59836e120e75bea7a5930c4d34aa10fe7a196a9
+
+build:
+  noarch: generic
```

Comment:
The package cannot be noarch due to the different scripts in `bin/`

Reply:
Updated in the recent commit